go.mod

```diff
@@ -1,16 +1,18 @@
 module github.com/aquasecurity/trivy-checks
 
-go 1.22.0
+go 1.22
+
+toolchain go1.22.0
```

Comment:
Need to keep in sync with Trivy due to https://github.com/aquasecurity/trivy/pull/6476#discussion_r1582593900